tests/jballerina-unit-test/src/test/resources/test-src/types/tuples/tuple_destructure_test.bal

```diff
@@ -109,3 +109,26 @@ function tupleDestructureTest9() returns [boolean, string, [int, float]] {
 
     return [a, b, c];
 }
+
+// TODO: This is temporary. Remove this once #42352 gets fixed
```

Comment:
Why does this have to be removed and how is this relevant to #42352?

Reply:
The idea is to have this function implemented to report an error aligning with previous behaviour.

Reply:
We don't have to remove this though? It's the negative test that has to be removed?

Reply:
Removed